modules/proxy-namespace-procedure.adoc

```diff
@@ -0,0 +1,42 @@
+
+[[red-hat-quay-proxy-namespace-procedure]]
+= Using {productname} to proxy a namespace in a registry
+
+The following procedure describes how you can use {productname} to proxy a namespace in a registry. This procedure is set up to proxy quay.io, which allows users to use `podman` to pull any public image from any namespace on quay.io.
+
+.Prerequisites
+
+*
+
+*
+
+.Procedure
+
+. In your Quay organization on the UI, for example, `cache-quayio`, click *Organization Settings* on the left hand pane.
+
+. In *Remote Registry* enter the name of the remote registry to be cached, for example, `quay.io`, and click *Save*.
++
+[NOTE]
+====
+* By adding a namespace to the *Remote Registry*, for example, `quay.io/<namespace>`, users in your organization will only be able to proxy from that namespace.
+* If you do not set a *Remote Registry Username* and *Remote Registry Password*, you cannot add one without removing the proxy cache and creating a new registry.
+* The default *Expiration* for cached images is set to 86400 seconds. This value can be adjusted. but is refreshed every pull. Un-refreshed images will disappear after the allotted time, but are still stored in Quay. The time in which an image is completely deleted, or garbage collected, depends on the *Time Machine* setting of your organization. The default time for garbage collection is 14 days unless otherwise specified.
```

Comment:
Oh and consider using the word `stale` rather than `un-refreshed`. In the future we'll have a field that will refer to staleness, so we might as well introduce the term now.

Reply:
Or actually `expired` would be better suited, since we're talking about tag expiration. And it's a term we already use.